modules/nginx-ingress-controller/variables.tf

```diff
@@ -0,0 +1,44 @@
+
+variable "eks_cluster_name" {
```

Comment:
This is called `cluster_name` in other modules. E.g. https://github.com/nuuday/terraform-aws-eks-addons/blob/master/modules/cluster-autoscaler/variables.tf